templates/taxi/car_list.html

```diff
@@ -10,8 +10,11 @@ <h1>Car list</h1>
           {{ car.model }} ({{ car.manufacturer.name }})
         </li>
       {% endfor %}
+      <br>
+      <a class="btn btn-primary" href="{% url 'taxi:car-create' %}">add new car</a>
     </ul>
   {% else %}
     <p>There are no cars in taxi</p>
   {% endif %}
+  
```

Comment:
Ensure there is a blank line at the end of the file as required by the checklist.